eo-maven-plugin/src/main/java/org/eolang/maven/footprint/Footprint.java

```diff
@@ -48,6 +50,13 @@ public interface Footprint {
      * @param content File content
      * @throws IOException In case of IO issues
      */
-    void save(String program, String ext, Scalar<String> content)
-        throws IOException;
+    void save(String program, String ext, Scalar<String> content) throws IOException;
+
+    /**
+     * Get list of saved regular files with ext.
```

Comment:
@volodya-lombrozo what do you call "regular file" here?

Reply:
@mximp Not a directory